platform/mds/mds/src/main/java/org/motechproject/mds/service/TableWriter.java

```diff
@@ -1,4 +1,4 @@
-package org.motechproject.mds.service.impl.csv.writer;
+package org.motechproject.mds.service;
```

Comment:
The pdf/csv importer/exporter classes are created as Spring beans directly in mds-entities bundle. Both of them extend an abstract class - AbstractMdsExporter, that uses the TableWriter interface and therefore it must be exported via OSGi. I've moved it to the service package, since I made impl.csv.writer package internal (that contains implementations of TableWriter)